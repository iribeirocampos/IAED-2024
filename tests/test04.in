p parque1 10 0.30 0.50 15.00
p "Parque 2" 20 0.30 0.50 15.00
p Parque3 10 0.25 0.40 20.00
p "Parque 2" 0 -0.10 1.00 0.90
p "Parque 4" 0 -0.10 1.00 0.90
p "Parque 5" 10 -0.10 1.00 0.90
p "Parque 6" 10 0.10 1.00 0.90
p "Parque 7" 10 0.10 0.50 5.00
p "Parque 8" 10 0.10 0.50 5.00
p "Parque 9" 10 0.10 0.50 5.00
p "Parque 10" 10 0.10 0.50 5.00
p "Parque 11" 10 0.10 0.50 5.00
p "Parque 12" 10 0.10 0.50 5.00
p "Parque 13" 10 0.10 0.50 5.00
p "Parque 14" 10 0.10 0.50 5.00
p "Parque 15" 10 0.10 0.50 5.00
p "Parque 16" 10 0.10 0.50 5.00
p "Parque 17" 10 0.10 0.50 5.00
p "Parque 18" 10 0.10 0.50 5.00
p "Parque 19" 10 0.10 0.50 5.00
p "Parque 20" 10 0.10 0.50 5.00
p "Parque 21" 10 0.10 0.50 5.00
p "Parque 22" 10 0.10 0.50 5.00
p "Parque 23" 10 0.10 0.50 5.00
p "Parque 24" 10 0.10 0.50 5.00
p
q
