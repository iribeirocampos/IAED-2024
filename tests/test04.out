Parque 2: parking already exists.
0: invalid capacity.
invalid cost.
invalid cost.
too many parks.
parque1 10 10
Parque 2 20 20
Parque3 10 10
Parque 7 10 10
Parque 8 10 10
Parque 9 10 10
Parque 10 10 10
Parque 11 10 10
Parque 12 10 10
Parque 13 10 10
Parque 14 10 10
Parque 15 10 10
Parque 16 10 10
Parque 17 10 10
Parque 18 10 10
Parque 19 10 10
Parque 20 10 10
Parque 21 10 10
Parque 22 10 10
Parque 23 10 10
